jboss/container/util/pkg-update/module.yaml

```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: jboss.container.util.pkg-update
+version: '1.0'
+description: "Perform a package manager update."
```

Comment:
s/Perform a package manager update/Perform a system update via the package manager/